compiler/test/runnable/testxmm2.d

```diff
@@ -73,10 +73,20 @@ void testeqne()
 
 /*****************************************/
 
+int4 testz4() { return [0,0,0,0]; }
+
+void test2()
+{
+    assert(testz4()[] == [0,0,0,0]);
```

Comment:
I don't have a machine to test it on :-)

Reply:
The 256 bit support in general is weak, I was deferring dealing with that until later.

Reply:
So does this PR expose new untested code generation to users of 256 bit SIMD in dmd? Because that sounds risky. (I don't have much experience with SIMD so excuse my superficial review)

Reply:
I added a test for it.

Reply:
Also for `~0`?